domains.txt

```diff
@@ -151,6 +173,7 @@ cpi-selection.co.uk
 cpsgroupuk.com
 creationrecruitment.co.uk
 creativepersonnel.co.uk
+crimson.co.uk
```

Comment:
Crimson is a fast-moving technology and recruitment company that specialises in the full range of IT support, consultancy, search and selection services.